ivre/xmlnmap.py

```diff
@@ -665,13 +666,75 @@ def change_rpcinfo(table):
             for proto, result in viewitems(protores)]
 
 
+def change_ms_sql_info(table):
+    """Adapt structured output generated by the "ms-sql-info" Nmap script. The
+structured output uses instances (hence, data) as keys, which is undesirable in 
```

Comment:
```suggestion
structured output uses instances (hence, data) as keys, which is undesirable in
```
Be careful with trailing whitespaces.

Reply:
Yes, thank you.